src/router.js

```diff
@@ -73,9 +67,13 @@ async function featureAvailable(store, networkSlug, feature) {
   const currentNetwork = networks.find(({ slug, id }) =>
     networkSlug ? slug === networkSlug : id === currentNetworkId
   )
-  const featureSelector = `feature_${feature.toLowerCase()}`
-  return typeof currentNetwork[featureSelector] === "string"
-    ? currentNetwork[featureSelector]
-    : // DEPRECATE fallback for old API response
-      networkCapabilityDictionary[currentNetwork[featureSelector]]
+  if (currentNetwork) {
+    const featureSelector = `feature_${feature.toLowerCase()}`
+    return currentNetwork[featureSelector]
+  } else {
+    setTimeout(
+      async () => await featureAvailable(store, networkSlug, feature),
+      100
+    )
```

Comment:
```suggestion
    await new Promise(resolve => setTimeout(resolve, 100))
    return featureAvailable(store, networkSlug, feature)
```

Reply:
Though: this would loop endlessly with a wrong networkSlug or id. Better move the loop up and check if the networks where loaded?

Reply:
ah yeah, good point